pom.xml

```diff
@@ -221,7 +221,9 @@
             <!-- The Auth Access Mode Service URI will have the ID to be checked appended onto the end -->
             <AUTH_ACCESS_SERVICE>http://0.0.0.0:${test.hauth.port}/access</AUTH_ACCESS_SERVICE>
             <AUTH_COOKIE_SERVICE>http://0.0.0.0:${test.hauth.port}/cookie</AUTH_COOKIE_SERVICE>
+            <AUTH_COOKIE_SERVICE_LABEL>UCLA Library</AUTH_COOKIE_SERVICE_LABEL>
```

Comment:
Could you put the "UCLA Library/Sinai Palimpsests at UCLA Library" values in properties, rather than writing them out three times each?

Reply:
okay, done